erp/export/schemas.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import Literal, Set
+
+
+@dataclass
+class OfficialSchema:
+    id: str
+    transport_station_presence: bool
+    transport_information: str
+    stationnement_presence: bool
+    stationnement_pmr: bool
+    stationnement_ext_presence: bool
+    stationnement_ext_pmr: bool
+    cheminement_ext_presence: bool
+    cheminement_ext_terrain_accidente: bool
+    cheminement_ext_plain_pied: bool
+    cheminement_ext_ascenseur: bool
+    cheminement_ext_nombre_marches: int
+    cheminement_ext_reperage_marches: bool
+    cheminement_ext_main_courante: bool
+    cheminement_ext_rampe: Literal["aucune", "fixe", "amovible"]
+    cheminement_ext_pente: Literal["aucune", "légère", "importante"]
+    cheminement_ext_devers: Literal["aucun", "léger", "important"]
```

Comment:
Argh, I realize we have accentuated letters in enum values 😢  Well, I suspect it's a bit late to update all this in the db…